### DISCUSSION TITLE
hi i am new bie

### DISCUSSION THREAD

**Discussion by recepozkan** [mafredri/asustor_as-6xxt#7] [General]
hi sorry i am new bie,
i need step by step install

**Comment by mafredri** [comment]
Hi @recepozkan, unfortunately I don't have one or the bandwidth to provide that level of support. Depending on which ASUSTOR model you have, installing Linux can be fairly easy or very involved, so Linux experience is basically a requirement.

You could take a look at e.g. https://www.reddit.com/r/asustor/comments/t3kh1t/newb_installing_ubuntu_linux_on_asustor_as5304t/ or https://www.reddit.com/r/asustor/comments/pj23sf/quick_guide_on_how_to_install_a_alternative_os_on/ to see if those are of any help to you.

**Reply by recepozkan** [reply]
thank you very much.
i have as-604t i wrong delete dom disk, i need orijinal dom image.
